assets/images/icon-sprite.svg

```diff
@@ -1 +1 @@
" d="M174.9 210.1c-2.1 0-3.8-1.8-3.8-3.9 0-2.1 1.8-3.8 3.9-3.8 2.1 0 3.8 1.8 3.8 3.9s-1.7 3.8-3.9 3.8zm.2-6.9c-1.7 0-3 1.3-3.1 3s1.3 3 3 3.1c1.7 0 3-1.3 3.1-3 0-1.7-1.4-3.1-3-3.1z"/><path class="st4" d="M178.2 207.6l2.8 2.9-1.5 1.5-2.8-2.9 1.5-1.5z"/></g><g><path class="st6" d="M149.1 226.3h-5.7m0 0v12.2m0 0h8m0 0v-9.6m0 0l-2.3-2.6m2.3 2.6h-2.5m0 0v-2.6"/><path class="st1" d="M151.4 228.9l-2.3-2.6v2.6"/><path class="st7" d="M139.6 227.6h8.4v5.4h-8.4v-5.4z"/><path class="st8" d="M142.3 230c0 .3-.1.5-.2.6-.2.2-.4.2-.7.2h-.2v.8h-.7v-2.5h.9c.3 0 .6.1.7.2s.2.4.2.7zm-1.1.3h.1c.1 0 .2 0 .2-.1.1-.1.1-.1.1-.2 0-.2-.1-.3-.3-.3h-.2l.1.6zm4 .1c0 .4-.1.7-.3 1-.2.2-.6.3-1 .3h-.8v-2.5h.9c.4 0 .7.1.9.3s.3.5.3.9zm-.7 0c0-.2 0-.4-.1-.5s-.2-.2-.4-.2h-.2v1.4h.2c.2 0 .4-.1.5-.2s0-.2 0-.5zm2.1 1.3h-.6v-2.5h1.5v.5h-.8v.5h.7v.5h-.7l-.1 1z"/></g><g><path class="st10" d="M180.3 226.3h-5.7m0 0v12.2m0 0h8m0 0v-9.6m0 0l-2.3-2.6m2.3 2.6h-2.4m0 0v-2.6"/><path class="st11" d="M182.6 228.9l-2.3-2.6v2.6"/><path class="st7" d="M170.8 227.6h8.4v5.4h-8.4v-5.4z"/><path class="st8" d="M173.5 230c0 .3-.1.5-.2.6-.2.2-.4.2-.7.2h-.2v.8h-.7v-2.5h.9c.3 0 .6.1.7.2s.2.4.2.7zm-1.1.3h.1c.1 0 .2 0 .2-.1.1-.1.1-.1.1-.2 0-.2-.1-.3-.3-.3h-.2l.1.6zm4 .1c0 .4-.1.7-.3 1-.2.2-.6.3-1 .3h-.8v-2.5h.9c.4 0 .7.1.9.3s.3.5.3.9zm-.7 0c0-.2 0-.4-.1-.5s-.2-.2-.4-.2h-.2v1.4h.2c.2 0 .4-.1.5-.2s0-.2 0-.5zm2.1 1.3h-.7v-2.5h1.5v.5h-.8v.5h.7v.5h-.7v1z"/></g><g><path class="st6" d="M155.1 322.3l-8-8m0 0l-8 8m8-8v16m0 0h8"/></g><g><path class="st15" d="M187.3 322.3l-8-8m0 0l-8 8m8-8v16m0 0h8"/></g><circle class="st13" cx="147.1" cy="354.3" r="8"/><g><path class="st4" d="M147.2 357.8H146l1.2-5.6h1.2l-1.2 5.6zm.2-7c0-.2.1-.4.2-.6.1-.1.3-.2.6-.2.2 0 .3 0 .4.1.1.1.2.2.2.4s-.1.4-.2.6c-.1.1-.3.2-.5.2s-.3 0-.4-.1c-.2-.1-.3-.2-.3-.4z"/></g><circle class="st13" cx="179.1" cy="354.3" r="8"/><g><path class="st4" d="M179 357.8h-.9l1.2-5.6h.9l-1.2 5.6zm.5-7c0-.2.1-.3.2-.5s.2-.2.4-.2c.3 0 .4.2.4.5 0 .2-.1.3-.2.5-.1.1-.2.2-.4.2-.1 0-.2 0-.3-.1 0-.2-.1-.3-.1-.4z"/></g><path class="st4" d="M147.1 378.3h8l-4 5.6-4 5.6-4-5.6-4-5.6"/><g><path class="st0" d="M14.2 791h32m-32 20h32m-32-.8V791m32 19.2V791m-16 12.2L46 792.1m-15.8 11.1l-15.9-11.1"/></g><g><path d="M80.3 791.1h32m-32 20h32m-32-.8v-19.2m32 19.2v-19.2m-16 12.2l15.8-11.1m-15.8 11.1l-15.8-11.1" fill="none" stroke="#f6f6f6" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"/></g><g><path class="st6" d="M139.1 403.1h16m-16 10h16m-16-.4v-9.6m16 9.6v-9.6m-8 5.9l7.9-5.5m-7.9 5.5l-7.9-5.5"/><path d="M172.2 403.2h16m-16 10h16m-16-.4v-9.6m16 9.6v-9.6m-8 6.1l7.9-5.6m-7.9 5.6l-7.9-5.6" fill="none" stroke="#f6f6f6" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"/></g><g><circle class="st0" cx="24.4" cy="399.7" r="6.7"/><path class="st0" d="M29.6 404.8l4.6 4.6"/></g><g fill="none" stroke="#f1f2f2" stroke-width="2" stroke-miterlimit="10"><circle cx="87.6" cy="399.7" r="6.7"/><path stroke-linecap="round" d="M92.7 404.8l4.6 4.6"/></g></svg>
```

Comment:
Probably best to leave this file alone and just run with the v3 version. Then change any references to this old one to the v3 one in the sixteens.